tests/test-helper.js

```diff
@@ -2,5 +2,7 @@ import resolver from './helpers/resolver';
 import {
   setResolver
 } from 'ember-qunit';
+// import { start } from 'ember-cli-qunit';
```

Comment:
If I recall, the tests didn't work with ember-cli-qunit@4.0.0-beta1. They apparently work with 4.0.0 now.